app/models/resource.rb

```diff
@@ -0,0 +1,2 @@
+class Resource < ApplicationRecord
```

Comment:
can you add a comment above this indicating the fields that the resource model has?